CHANGELOG.md

```diff
@@ -1,6 +1,24 @@
 StorIO Change Log
 ==========
 
+## Version 1.12.0
+
+_2016_12_7_
+
+* Support for **AutoValue** and **Kotlin data classes** in StorIOSQLiteProcessor and StorIOContentResolverProcessor. Many thanks to [@geralt-encore](https://github.com/geralt-encore)!
```

Comment:
It's not only about data classes. It can be used with a regular Kotlin class too.